packages/engine/src/transform/systems/TransformSystem.ts

```diff
@@ -303,7 +304,8 @@ export default async function TransformSystem(world: World) {
         world.dirtyTransforms[getOptionalComponent(entity, LocalTransformComponent)?.parentEntity ?? 0] ||
         world.dirtyTransforms[getOptionalComponent(entity, ComputedTransformComponent)?.referenceEntity ?? 0] ||
         hasComponent(entity, ComputedTransformComponent)
-      if (makeDirty) world.dirtyTransforms[entity] = true
+
+      world.dirtyTransforms[entity] = makeDirty
```

Comment:
`world.dirtyTransfoms` is defined as `Record<Entity, true>`, and we remove items from that record via `delete world.dirtyTranforms[eid]`

If we are going to allow this record to contain `false`, we should also change it's definition, and change all the `delete` operations to instead set the value to `false`.

Reply:
Hmm, maybe it'd be best to make that into its own PR rather than shoehorn it into a simple animation fix then, assuming this is worth it for performance.